ui/lib/core/addon/components/alert-banner.js

```diff
@@ -13,36 +11,28 @@ import layout from '../templates/components/alert-banner';
  * ```
  *
  * @param {String} type=null  - The banner type. This comes from the message-types helper.
- * @param {String} [secondIconType=null] - If you want a second icon to appear to the right of the title. This comes from the message-types helper.
- * @param {Object} [progressBar=null] - An object containing a value and maximum for a progress bar. Will be displayed next to the message title.
+ * @param {String} [bannerType=alert] - Defaults to 'alert', can be used to specify an alert banner's test selector.
```

Comment:
I think we typically try to avoid adding args just for test selecting - I wonder if we can just hard code 'alert' in the template. I don't even see a test that uses `data-test-alert-banner="license"` which is the only other `bannerType` I found - so `@bannerType` can probably be removed all together (`license-banners.hbs` too)

Reply:
Good catch. Amended